contracts/cw-core/src/contract.rs

```diff
@@ -705,6 +737,34 @@ pub fn query_cw20_balances(
     to_binary(&balances)
 }
 
+pub fn query_list_sub_daos(
+    deps: Deps,
+    start_after: Option<String>,
+    limit: Option<u32>,
+) -> StdResult<Binary> {
+    let start_at = start_after
+        .map(|addr| deps.api.addr_validate(&addr))
+        .transpose()?;
+
+    let subdaos = cw_paginate::paginate_map(
+        deps,
+        &SUBDAO_LIST,
+        start_at.as_ref(),
+        limit,
+        cosmwasm_std::Order::Ascending,
+    )?;
+
+    let subdaos = subdaos
+        .into_iter()
+        .map(|(address, charter)| SubDao {
+            addr: address.into_string(),
+            charter,
+        })
+        .collect();
+
+    to_binary(&ListSubDaosResponse { subdaos })
```

Comment:
How would you feel about returning `to_binary(subdaos)` here? Would match the other queries and what you have in the `schema.rs` file above.

Reply:
updated